### PR TITLE
Responsive and few improvements

### PR DESCRIPTION
**Pull request by LukyVj**

### CODE REVIEW

File: src/templates/layout.html [hunk 1]
```diff
@@ -9,12 +9,14 @@
...
+  <link rel="stylesheet" href="style/tomorrow-night-eighties.css">
 </head>
 <body>
   {{{ content }}}
 
   <script src="https://cdn.firebase.com/js/client/2.2.9/firebase.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/highlight.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
+  <script>hljs.initHighlightingOnLoad();</script>
```
Comment (Haroenv) [inline src/templates/layout.html]: Why is this here and not in `main.js`?

Comment (LukyVj) [inline src/templates/layout.html]: Just followed the doc, but yeah, it could be inside the `main.js`, good call !

File: src/public/style/style.css [hunk 2]
```diff
@@ -300,4 +301,68 @@ message .-message p {
 /* Some utils */
 .avatar {
   border-radius: 10px;
+}
+
+
+/* Style code blocks */
+pre {
+    font-family: "Courier 10 Pitch", Courier, monospace;
```
Comment (Haroenv) [inline src/public/style/style.css]: Why so many different font stacks?

File: src/public/style/style.css [hunk 3]
```diff
@@ -300,4 +301,68 @@ message .-message p {
...
+    font-size: 95%;
+    line-height: 140%;
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
+    white-space: -o-pre-wrap;
+}       
+
+code {
+    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
```
Comment (Haroenv) [inline src/public/style/style.css]: This is my preferred one :)

File: src/public/style/style.css [hunk 4]
```diff
@@ -300,4 +301,68 @@ message .-message p {
...
+    letter-spacing: -1.159999996423721px;
+    -moz-tab-size: 4;
+    -o-tab-size: 4;
+    tab-size: 4;
+    text-align: left;
+    text-rendering: optimizeLegibility;
+    -webkit-box-shadow: inset 0 0 0 .1em #eff0f2;
+    box-shadow: inset 0 0 0 .1em #eff0f2;
+    border-radius: 4.80000019073486px 4.80000019073486px 4.80000019073486px 4.80000019073486px;
+    font: normal normal 200 normal 16px/28px Consolas, 'Andale Mono WT', 'Andale Mono', 'Lucida Console', 'Lucida Sans Typewriter', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Liberation Mono', 'Nimbus Mono L', Monaco, 'Courier New', Courier, monospace;
```
Comment (Haroenv) [inline src/public/style/style.css]: here is another font stack?

File: src/public/scripts/main.js [hunk 5]
```diff
@@ -1,70 +1,101 @@
...
+
+
+  // create message template
+  var template = '<message data-time-stamp="10-11-2015">' +
+  '<div class="user">' +
+  '  <span class="-avatar"> <img src="https://avatars.githubusercontent.com/' + newPost.data.author + '" alt="avatar user" /> </span>' +
+  '  <span class="-name"> '+ newPost.data.author +' </span>' +
+  ' </div>' +
+  ' <div class="-message">' +
+  ' <p>'+ marked(newPost.data.body) +'</p>' +
```
Comment (Haroenv) [inline src/public/scripts/main.js]: as said, this `<p>` needs to go

File: src/public/style/style.css (hunk 4 above)
Comment (LukyVj) [inline src/public/style/style.css]: Keen eyes dude, i got them from Medium, and I think that if medium do it, it could not be bad to follow them :) 

By the way, I reduced the numbers of font declarations!

**Comment by Haroenv** [comment]
LGTM
